source/mir/date.d

```diff
@@ -572,35 +574,37 @@ struct YearMonth
     }
 
     ///
-    this(short year, Month month) @safe pure nothrow @nogc
+    @ReflectDoc!"SIL"("Construct a YearMonth from a year and a month.", reflectUnittest!"SIL"(`import YearMonth from xenon.date
+enforce(YearMonth(2020, 10).year == 2020, "Should match")
+enforce(YearMonth(2020, 10).month == 10, "Should match")`))
+    this(short year, ushort month) @safe pure @nogc
     {
+        static immutable exc = new Exception("Month out of bounds [1, 12]");
+        if (1 > month || month > 12)
+            throw exc;
         this.year = year;
-        this.month = month;
+        this.month = cast(Month)month;
     }
 
     ///
-    version (mir_test)
-    @safe unittest
-    {
-        auto ym = YearMonth(2000, Month.dec);
-    }
-
-    ///
-    this(short year, Quarter quarter, AssumePeriod assumePeriod = AssumePeriod.begin) @safe pure nothrow @nogc
+    @reflectIgnore!"SIL" // clashes with ushort version
+    this(short year, Month month) @safe pure nothrow @nogc
```

Comment:
@9il This might have been a typo on my part. I think there was mean to be a version that takes a `Quarter` as an input.